From f2dc0854251c7739a370856beef1133b5f1294ac Mon Sep 17 00:00:00 2001
From: Noam Yorav-Raphael <noamraph@gmail.com>
Date: Wed, 20 Nov 2024 00:01:49 +0200
Subject: [PATCH] Fix SyntaxWarnings

Currently when running ponysay, you get this warning twice:

```
/nix/store/.../bin/ponysay/balloon.py:43: SyntaxWarning: invalid escape sequence '-'
```

It is caused by a docstring containing twice the phrase `\-directional`.

The fix is just to make it a raw string (replace `'''` with `r'''`).
---
 src/balloon.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/balloon.py b/src/balloon.py
index 0f1909e4d..a938ece01 100644
--- a/src/balloon.py
+++ b/src/balloon.py
@@ -40,7 +40,7 @@ class Balloon():
     '''
     
     def __init__(self, link, linkmirror, linkcross, ww, ee, nw, nnw, n, nne, ne, nee, e, see, se, sse, s, ssw, sw, sww, w, nww):
-        '''
+        r'''
         Constructor
         
         @param  link:str        The \-directional balloon line character
